norsk-henvisningsstandard-for-rettsvitenskapelige-tekster.csl

```diff
@@ -286,8 +286,12 @@
             <text prefix=", " macro="title"/>
             <text prefix=" " font-style="italic" variable="container-title"/>
             <text prefix=" " macro="issued"/>
-            <text prefix=" " suffix=" " term="page" form="short"/>
-            <text variable="page"/>
+            <choose>
+              <if variable="page" match="any">
+                <text prefix=" " suffix=" " term="page" form="short"/>
+                <text variable="page"/>
+              </if>
+            </choose>
```

Comment:
```suggestion
            <group delimiter=" " prefix=" ">
              <text term="page" form="short"/>
              <text variable="page"/>
            </group>
```

Reply:
I had a slight hunch that there was a simpler solution. Thanks for spelling it out, so that I can learn!